capstone/src/main/java/com/google/univiz/SuggestionDataApi.java

```diff
@@ -3,13 +3,13 @@
 import java.util.List;
 
 /**
- * Communicates directly with the REST API to get a list of college suggestions related to a client
- * query
+ * Communicates directly with the data source to get a list of college suggestions related to a
+ * client query
  */
 public interface SuggestionDataApi {
   /**
-   * Takes a list of college names and returns a list of SuggestionData objects, where each object
-   * is unique to each college name providede in the first list
+   * Takes a college name and returns a list of SuggestionData objects, where each object is unique
+   * to each college name provided in the first list
```

Comment:
what does "provided in the first list" mean, in this case?

Reply:
updated!